### PR TITLE
Use coala bears master

### PR DESCRIPTION
**Pull request by jayvdb**

### CODE REVIEW

File: .coafile [hunk 1]
```diff
@@ -31,13 +31,13 @@ max_line_length = 120
...
 
-[html]
+[all.html]
 bears = HTMLLintBear
 files = **.html
 htmllint_ignore = optional_tag, concerns_separation
 ignore = public/**
 
-[javascript]
+[all.javascript]
```
Comment (Monal5031) [inline .coafile]: preferred quotation in section `all` is `'`, maybe that will affect JS's usage of `"`

**Comment by jayvdb** [comment]
Netlify has been failing here for a while.

Migrating this to GitLab will work around that, as GitLab Pages will deploy to the forks CI, which can have GH and GL tokens, and is a bit easier to debug.
 
https://github.com/coala/community/pull/202 is part of that

**Comment by jayvdb** [comment]
ack https://github.com/coala/community/pull/203/commits/2b1ed43cd51cdda0470cfa3b36caab922a1d10d6 https://github.com/coala/community/pull/203/commits/e053ed7d7a709e871d4091477dc69e69ea22dfcb https://github.com/coala/community/pull/203/commits/ece5a115fc2cf99fa816e054a4694e36d0fc456d

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**